internal/store/users.go

```diff
@@ -0,0 +1,62 @@
+package store
+
+import (
+	"fmt"
+
+	"github.com/lib/pq"
+)
+
+// ErrExists is returned if a unique record already exists.
+var ErrExists = fmt.Errorf("a record already exists")
+
+const pgExists = "23505"
+
+// User holds information about a user such as their id, username, and hashed
+// password.
+type User struct {
```

Comment:
Why is there JSON on a store type?

Reply:
Because it would be dumb to create a type in server that is exactly the same just with tags.